### DISCUSSION TITLE
The date of weekly meeting and the meeting room

### DISCUSSION THREAD

**Discussion by TinhToQuang** [Devtoberfest2021OpenSAPTeamUp/OpenSAP-certification-API#9] [General]
Hi @Devtoberfest2021OpenSAPTeamUp/opensap-courses-list-certification-api ,
Can we discuss about:
- Which appropriate date/time we can discuss and share the idea about this project?
- Which social media we use for meeting together? (Skype, Slack, Whatsapp, etc ...)?

**Comment by AmauryVanEspen** [comment]
A MVP CAP app is available
